Makefile

```diff
@@ -284,6 +294,10 @@ mariadb: namespace mariadb_prep ## installs the operator, also runs the prep ste
 	$(eval $(call vars,$@,mariadb))
 	oc apply -f ${OPERATOR_DIR}
 
+.PHONY: mariadb_deploy_validate ## checks that mariadb was properly deployed. Set KEYSTONE_KUTTL_DIR to use assert file from custom repo.
+mariadb_deploy_validate: input namespace
+	kubectl-kuttl assert -n ${NAMESPACE} ${MARIADB_KUTTL_DIR}/mariadb_deploy/01-assert.yaml --timeout 180
```

Comment:
should the assert file for all operators for the sample always be in something like ${XYZ_KUTTL_DIR}/../common/assert_sample_deployment.yaml so that _deploy_validate in install_yamls always can reference this?

Reply:
That's a good idea to have a consistent naming. I'll update the keystone PR, and since the mariadb PR one was merged already, I'll open a new one.

Reply:
Done, also created a new PR for mariadb-operator: https://github.com/openstack-k8s-operators/mariadb-operator/pull/49